tritium-lib/src/test/java/com/palantir/tritium/proxy/InvocationEventProxyTest.java

```diff
@@ -74,21 +75,18 @@ public void testInstrumentPreInvocation() throws Throwable {
         InvocationEventHandler<InvocationContext> testHandler = new SimpleHandler();
         InvocationEventProxy proxy = createTestProxy(testHandler);
 
-        Object result1 = proxy.handleInvocation(this, getToStringMethod(), EMPTY_ARGS);
-
-        assertThat(result1).isNotNull();
-        assertThat(result1.toString()).isEqualTo("test");
+        assertThat(proxy.handleInvocation(this, getToStringMethod(), EMPTY_ARGS))
+                .isNotNull().extracting(Object::toString).isEqualTo("test");
 
         Object result2 = proxy.handlePreInvocation(this, getToStringMethod(), EMPTY_ARGS);
-        assertThat(result2).isNotNull();
         assertThat(result2).isInstanceOf(DefaultInvocationContext.class);
-        assertThat(result2.toString()).contains(InvocationEventProxyTest.class.getName());
+        assertThat(Objects.requireNonNull(result2).toString()).contains(InvocationEventProxyTest.class.getName());
 
         InvocationContext context = proxy.handlePreInvocation(this, getToStringMethod(), EMPTY_ARGS);
-        assertThat(context).isNotNull();
-        assertThat(context.toString()).contains("startTimeNanos");
-        assertThat(context.toString()).contains("instance");
-        assertThat(context.toString()).contains("method");
+        assertThat(Objects.requireNonNull(context).toString())
```

Comment:
assertj can take care of the null check
```suggestion
        assertThat(context).asString()
```